.circleci/config.yml

```diff
@@ -63,6 +63,18 @@ jobs:
                   else
                       touch test_preparation/examples_test_list.txt
                   fi
+            - run: |
+                  if [ -f filtered_test_list_torch_and_tf.txt ]; then
+                      mv filtered_test_list_torch_and_tf.txt test_preparation/filtered_test_list_torch_and_tf.txt
+                  else
+                      touch test_preparation/filtered_test_list_torch_and_tf.txt
+                  fi
+            - run: |
+                  if [ -f filtered_test_list_torch_and_flax.txt ]; then
+                      mv filtered_test_list_torch_and_flax.txt test_preparation/filtered_test_list_torch_and_flax.txt
+                  else
+                      touch test_preparation/filtered_test_list_torch_and_flax.txt
+                  fi
```

Comment:
Let's put everything in one report for cross tests (named filtered_crossed_test_list for instance)